Frontend/src/test/components/ui/checkbox.test.jsx

```
@@ -0,0 +1,63 @@
+vi.mock("@radix-ui/react-checkbox", () => ({
+  __esModule: true,
+  Root: React.forwardRef(({ className, checked, disabled, onCheckedChange, children, ...props }, ref) => (
+    <div
+      ref={ref}
+      role="checkbox"
+      aria-checked={checked ? "true" : "false"}
+      aria-disabled={disabled ? "true" : "false"}
+      className={className}
+      onClick={() => !disabled && onCheckedChange && onCheckedChange(!checked)}
+      {...props}
+    >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "checkbox" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "checkbox".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)